io_directory_example/settings/lens_system3_config.yml

```diff
@@ -17,9 +17,14 @@ model:
   lens_light:
     - 'SERSIC_ELLIPSE'
     - 'SERSIC_ELLIPSE'
+    - 'SERSIC_ELLIPSE'
+    - 'SERSIC_ELLIPSE'
+  lens_light_band_index: [0,0,1,1]
```

Comment:
Add spaces after comma.

Reply:
Change `lens_light_band_index` to `lens_light_band_indices`, then it's clearer that this value has to be something like a `list`, and not just a single `float`.